worlds/messenger/Constants.py

```diff
@@ -0,0 +1,153 @@
+# items
+# listing individual groups first for easy lookup
+NOTES = [
+    "Key of Hope",
+    "Key of Chaos",
+    "Key of Courage",
+    "Key of Love",
+    "Key of Strength",
+    "Key of Symbiosis"
+]
+
+PROG_ITEMS = [
+    "Wingsuit",
+    "Rope Dart",
+    "Ninja Tabi",
+    "Power Thistle",
+    "Demon King Crown",
+    "Ruxxtin's Amulet",
+    "Fairy Bottle",
+    "Sun Crest",
+    "Moon Crest",
+    # "Astral Seed",
+    # "Astral Tea Leaves",
+]
+
+PHOBEKINS = [
+    "Necro",
+    "Pyro",
+    "Claustro",
+    "Acro"
+]
+
+USEFUL_ITEMS = [
+    "Windmill Shuriken"
+]
+
+# item_name_to_id needs to be deterministic and match upstream
+ALL_ITEMS = [
+    *NOTES,
+    "Windmill Shuriken",
+    "Wingsuit",
+    "Rope Dart",
+    "Ninja Tabi",
+    # "Astral Seed",
+    # "Astral Tea Leaves",
+    "Candle",
+    "Seashell",
+    "Power Thistle",
+    "Demon King Crown",
+    "Ruxxtin's Amulet",
+    "Fairy Bottle",
+    "Sun Crest",
+    "Moon Crest",
+    *PHOBEKINS,
+    "Power Seal",
+    "Time Shard"  # there's 45 separate instances of this in the client lookup, but hopefully we don't care?
+]
+
+# locations
+# the names of these don't actually matter, but using the upstream's names for now
+# order must be exactly the same as upstream
+ALWAYS_LOCATIONS = [
+    # notes
+    "Key of Love",
+    "Key of Courage",
+    "Key of Chaos",
+    "Key of Symbiosis",
+    "Key of Strength",
+    "Key of Hope",
+    # upgrades
+    "Wingsuit",
+    "Rope Dart",
+    "Ninja Tabi",
+    "Climbing Claws",
+    # quest items
+    "Astral Seed",
+    "Astral Tea Leaves",
+    "Candle",
+    "Seashell",
+    "Power Thistle",
+    "Demon King Crown",
+    "Ruxxtin's Amulet",
+    "Fairy Bottle",
+    "Sun Crest",
+    "Moon Crest",
+    # phobekins
+    "Necro",
+    "Pyro",
+    "Claustro",
+    "Acro",
```

Comment:
```suggestion
    "Acro"
```
as to your reply to el-u's comment :D

I do think trailing comma is a good idea because it's less noisy (fewer merge conflicts) when appending to a list. But most important is consistency.